libpod/options.go

```diff
@@ -850,3 +814,52 @@ func WithGroups(groups []string) CtrCreateOption {
 		return nil
 	}
 }
+
+// WithUserVolumes informs libpod that the container has user-added volumes
+// It is used to for triggering hooks that check for the presence of volume
+// mounts
```

Comment:
You'll want trailing periods on these sentences for the rendered godocs (e.g. [here][1]).

[1]: https://godoc.org/github.com/projectatomic/libpod/libpod#WithCgroupNSFrom